capstone-app/src/app/modules/photos/components/image-container/image-container.component.scss

```diff
@@ -1,13 +1,108 @@
 .flex-container {
   display: flex;
   flex-wrap: wrap;
+  font-family: Georgia;
+  letter-spacing: 1.5px;
   margin-left: 10%;
   margin-right: 10%;
+  width: 80%;
+}
+
+.flex-container .picture {
+  border-radius: 3px;
+  box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.05);
+  display: inline-block;
+  opacity: 0.99;
+  overflow: hidden;
+  position: relative;
+  text-align: left;
+}
+
+.flex-container .picture:before {
+  background: -webkit-gradient(
+    linear,
+    left top,
+    left bottom,
+    from(rgba(0, 0, 0, 0)),
+    to(rgba(0, 0, 0, 0.8))
```

Comment:
I see a lot of styles here that can be nested
.flex-container {
  .picture{
    img{
    }
  }

  .picture:hover{}
 // etc.
}